README.md

```diff
@@ -41,11 +41,26 @@ For docs and info see the [wiki](https://github.com/jasync-sql/jasync-sql/wiki).
   <artifactId>jasync-mysql</artifactId>
   <version>2.1.7</version>
 </dependency>
+
 <!-- postgresql -->
 <dependency>
-  <groupId>com.github.jasync-sql</groupId>
-  <artifactId>jasync-postgresql</artifactId>
-  <version>2.1.7</version>
+    <groupId>com.github.jasync-sql</groupId>
+    <artifactId>jasync-postgresql</artifactId>
+    <version>2.1.7</version>
+</dependency>
+
+<!-- r2dbc-mysql -->
+<dependency>
+    <groupId>com.github.jasync-sql</groupId>
+    <artifactId>jasync-r2dbc-mysql</artifactId>
+    <version>2.1.7</version>
+</dependency>
+
+<!-- r2dbc-pool-->
```

Comment:
jasync pool is not r2dbc pool. I suggest to omit that as it will just cause confusion.

Reply:
Okay, thank you.